Documentation/sources/content-card-ui/public-classes/aepui.md

```diff
@@ -0,0 +1,96 @@
+# Sealed Interface - AepUI
+
+The `AepUI` interface represents a UI component that can be rendered using the AEP compose UI library. In the initial release of the AEP compose UI library, the `SmallImageUI` implements an `AepUI`  to display a Small Image Template. The `AepUI` associates with a specific template and state to ensure consistency when rendering and managing UI components.
```

Comment:
Can we rephrase `in initial release...` so we can keep adding to it as more templates are release. How about
```
The AEP compose UI currently supports rendering the following UI templates:
1. `SmallImageUI` which renders `Small Image template` 
```